datavault/__init__.py

```diff
@@ -318,158 +323,78 @@ def getTags(self, sessions, datasets):
         return sessTags, dataTags
 
 class Dataset(object):
-    def __init__(self, session, name, title=None, num=None, create=False, independents=[], dependents=[]):
+    """
+    This object basically takes care of listeners and notifications.
+    All the actual data or metadata access is proxied through to a
+    backend object.
+    """
```

Comment:
It's helpful (and part of the style guide) to list the important attributes of a class.